nominal/core/client.py

```diff
@@ -963,6 +965,36 @@ def get_data_review(self, rid: str) -> DataReview:
         response = self._clients.datareview.get(self._clients.auth_header, rid)
         return DataReview._from_conjure(self._clients, response)
 
+    def create_event(
+        self,
+        name: str,
+        asset_rids: Sequence[str],
```

Comment:
Don't need to change anything here, but just noting this is another example of what @stefanv previously pointed out with the annoying fact that a `str` _is a_ `Sequence[str]` but is generally invalid as input - we should probably holistically check that across the whole library in a diff PR